pkg/summarizer/summary.go

```diff
@@ -968,8 +980,17 @@ func allLinkedIssues(rows []*statepb.Row) []string {
 	return linkedIssues
 }
 
+// columnCellMetrics aggregates columnar and cellular metrics as a struct
+type columnCellMetrics struct {
```

Comment:
nit: wondering if there's a better name for this and the associated variables? (gridMetrics seems great, but it's also the method name, and `metrics` is too generic/confusable with the module metrics. :( Maybe 'gridStats' or something?

@chases2 any suggestions here?

Reply:
I was thinking about gridMetrics too :) but didn't want to create confusion... `gridStats` actually sounds pretty good too.

Reply:
changed the struct name to `gridStats` and variable name to metrics. PTAL